wporg-gp-translation-events.php

```diff
@@ -240,10 +211,15 @@ public function event_status_transition( string $new_status, string $old_status,
 			return;
 		}
 		if ( 'publish' === $new_status && ( 'new' === $old_status || 'draft' === $old_status ) ) {
-			$attendee_repository = new Attendee_Repository();
-			$current_user_id     = get_current_user_id();
-			if ( ! $attendee_repository->is_attending( $post->ID, $current_user_id ) ) {
-				$attendee_repository->add_attendee( $post->ID, $current_user_id );
+			$event_id            = $post->ID;
+			$user_id             = get_current_user_id();
```

Comment:
@psrpinto and I talked about this and that it should probably rather be `$user_id = $post->post_author`

Reply:
So, if:
- User1 creates a draft event.
- User2 publishes this event.
- The user1 will be set as attendee and as host.

It makes sense to me. I will update it in https://github.com/WordPress/wporg-gp-translation-events/pull/200.

Reply:
@amieiro actually made exactly that change in https://github.com/WordPress/wporg-gp-translation-events/pull/200/files/ca6c64390702d6e46f4302aa5cd835e202b6b010..5bd799f4757f54dfd871ad80aa98b1a8ae412512

Reply:
Thanks, @amieiro!